Remotion/Data/DomainObjects/Mapping/ClassDefinitionCollectionFactory.cs

```diff
@@ -53,17 +53,17 @@ public ClassDefinition[] CreateClassDefinitionCollection (IEnumerable<Type> type
       return classDefinitions.Values.ToArray();
     }
 
```

Comment:
I think we should expand this to TypeDefCollectionFactory and accept that the logic handles interfaces as well.

Reply:
I would do this in a future task that requires it instead of pushing this change into this task.

Reply:
pelase add a R2I todo then